### DISCUSSION TITLE
v6.1.0

### DISCUSSION THREAD

**Discussion by chuot** [chuot/rdio-scanner#75] [Announcements]
## Version 6.1

- Calls now support patched talkgroups.
- New search patched talkgroups option which is disabled by default.
- Talkgroups and units are now stored in their own database table.
- New units CSV importer.
- Fix blacklisted talkgroups being created anyway when autopopulate is enabled.
- Fix compatibility with mysql/mariadb (default sqlite is still recommended).

<hr /><em>This discussion was created from the release <a href='https://github.com/chuot/rdio-scanner/releases/tag/v6.1.0'>v6.1.0</a>.</em>

**Comment by atlex** [comment]
Hi @chuot 
Thank you for the rdio-scanner!

Dirwatch doesn't work for me. I can't ingest any files. 
v6.1.1, Ubuntu
Directory: /home/pi/recs
Extension: mp3
Type: SDR Trunk

Rdio Scanner v6.1.1
----------------------------------
2022/01/05 19:36:41 server started
2022/01/05 19:36:41 base folder is /home/pi/rdio-scanner
2022/01/05 19:36:41 main interface at http://host1:3000
2022/01/05 19:36:41 admin interface at http://host1:3000/admin

**Reply by chuot** [reply]
Please confirm that the dirwatch is monitoring a local folder, not a network mounted one.

**Reply by chuot** [reply]
If it's local, then attach (or send by email) a MP3 file generated by SDRTrunk and I will try to understand why it isn't working for you.

**Reply by atlex** [reply]
Yes. it's a local folder.

**Reply by atlex** [reply]
@chuot I have 2 types of files. I checked a Comment section of mp3's meta.

1: Date:2022-01-05 20:06:23.067;System:155;Name:900;Decoder:NBFM;Frequency:155900000;SNRVAL:25.5;
This one works fine. There are no issues with ingestion.

2: Date:2022-01-05 21:17:15.032;System:12;Site:55;Name:TRAFFIC;Decoder:DMR;Frequency:856123000;
It doesn't work. It's impossible to ingest :(

I don't see in ParseSdrTrunkMeta "server/parsers.go:29" any code for "Site". 
May be this is a problem?

**Reply by chuot** [reply]
That is what I want to test indeed. Please send me by email the MP3 that doesn't work.

**Reply by chuot** [reply]
Please test with v6.1.2, it should work now.

**Reply by atlex** [reply]
Yes! It works in v6.1.2. Thank you!

**Comment by noamlivne** [comment]
Hi @chuot,
I will check the issue I had till 6.0.7 with the Blacklisted Talkgroups ASAP. Thank you very much. 
In the meantime, I installed version 6.1.2 and Imported my Config file I saved from version 6.07, using the Tools tab.
After the Import, all the Talkgroups were somehow arranged according to their ID numbers, and not according to the order I manually set at 6.0.7 (using the six dots handle on the left of each Talkgroup).
The Sort Talkgroups configuration in the Options tab is still disabled (and was also disabled in 6.0.7).
Is this a bug, or should I manually rearrange the Talkgroups again?

**Reply by noamlivne** [reply]
I got a tip from another user: I started from scratch with 6..1.2, copied the .db file from 6.0.7, and now the Talkgroups are in the order I had them before! 
So I guess that is a required step between 6.0.7 to 6.1.2. I guess that from now on - for future versions, I can start again using the Backup option.

**Reply by chuot** [reply]
The ordering is now exported in the configuration backup since v6.1.0. You should be fine if you use a backup that has been done on a version >= 6.1.0.

**Comment by AvalancheAddict** [comment]
I upgraded from 6.0.7 to 6.1.2 & simply copied the database over and all seems to be good.  So as of right now I am running 6.1.2 with one 6.0.7 instance upstreaming into it and that continues to work well.  The only problem I have is that before and after the upgrade, whenever I try to delete an unused System, they appear to go away but they always come back, but that's not a big deal for me.    On another note, is anyone running this on a cloud based Windows machine, and if so who are you using as the provider?  I would really like to move my head end instance to the cloud and then upstream into that and use the cloud based one as the "server", especially since my ISP doesn't allow hosting "servers", and I'm sure it would be more reliable.

**Reply by chuot** [reply]
Reappearing systems have been fixed starting from 6.1. Lets see if there still reappearing when you'll update your 6.0 instance to 6.1. Keep up posted, thanks.
